Cartography/LayoutGuideProxy.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LayoutGuideProxy.swift
+//  Cartography
+//
+//  Created by Vitor Travain on 11/10/17.
+//  Copyright © 2017 Robert Böhnke. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+import UIKit
+
+@available(iOS, introduced: 9.0)
+@available(tvOS, introduced: 9.0)
+public final class LayoutGuideProxy: SupportsPositioningLayoutProxy {
```

Comment:
`UILayoutGuide` doesn't support `firstBaseline`/ `lastBaseline`, so this should not conform to `SupportsBaselineLayoutProxy` (super protocol).

Reply:
I'll be patching up that. Thanks @inamiy

Reply:
Also, `owningView` has been implicitly tested. I think the missing test cases are for `safeAreaLayoutGuide` which I'll also add. I'll also rename `LayoutElement` to `LayoutItem`.

I'll try to figure out something about the IUOs.